src/init/features/dataconnect/index.ts

```diff
@@ -96,6 +99,10 @@ export async function doSetup(setup: Setup, config: Config): Promise<void> {
       waitForCreation: false,
     });
   }
+  logger.info("");
```

Comment:
Did you forget to remove this?

Reply:
Nope, added this to space out the next message a little better so it is more visible